building/tracks/syllabus/next-exercises.md

```diff
@@ -0,0 +1,27 @@
+# Syllabus - Next Exercises
+
+Having implemented your first concept exercise, the next few exercises should build on that.
+
+We like to have three to five exercises that have no other prerequisites other than the `basics` concept.
+
+Good options are the core primitives or data types of your language.
+E.g. booleans, basic numeric types, strings, dates, and atoms.
```

Comment:
Is "dates" really a good option for something that only depends on "basics"? :thinking:

Reply:
I think you're right. This was my suggestion, but I think dates should indeed be removed here.

Reply:
Agreed! I'll remove it.